.github/remark.yaml

```diff
@@ -8,6 +8,8 @@ plugins:
 # General formatting
   - - remark-lint-emphasis-marker
     - '*'
+  - - remark-lint-no-undefined-references
```

Comment:
Could we still check the references by installing the GFM module or so?

Reply:
Can we enable this again now that GFM is enabled?

Reply:
even with GFM, this still results in an error. I was unable to figure out how to configure remark to enable this rule but ignore checkboxes.

Reply:
Bummer. I'll have a second look at it next week.

Reply:
It just works. I just activated remark-lint-no-undefined-references again and it works.